wiki_docs/Create How_GTM_works

```diff
@@ -0,0 +1,47 @@
+User can add or manage the AdWords, Google analytics, Flood-light and non-Google tags using the GTM.
+
+How it works
+  Followings are main components of the GTM:
+
+Account
+Each account can contains multiple containers
+  "One account per company/organization"
+
+Container
+A container includes the tracking tags (e.g. Google Analytics, AdWords Conversion Tracking tags) that you want to use on the website.
+  "One container for one website"
+
+Tags
+Tags are snippets of code on your site that collect data
+  "A tag must have at least one rule in order to fire"
```

Comment:
@amitkumarj441: I have a lot of comments. Let me know if anything is unclear. Thanks for getting this page started!